styles2.css

```diff
@@ -0,0 +1,158 @@
+:root {
+  --text-color1: #172b4d;
+  --text-green: #36b37f;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: sans-serif;
+  font-style: normal;
+}
+
+span {
+  color: var(--text-green);
+}
+
+#smallScreen {
+  display: flex;
+  justify-content: space-around;
+}
+
+#smallScreen > span {
+  color: var(--text-color1);
+  font-size: 20px;
+  line-height: 28px;
+}
+
+#main {
+  width: 100vw;
+  height: 100vh;
+  background-image: url(Header-llustration-desktop.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+}
+
+nav {
+  display: none;
+}
+
+a {
+  text-decoration: none;
+  color: var(--text-color1);
+}
+
+div.headline {
+  height: 80%;
+  width: 80%;
+  margin-left: 10%;
+  margin-top: 5%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+p {
+  width: 80%;
+  text-align: center;
+  margin: 0 auto;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+h2 {
+  text-align: center;
+  color: var(--text-color1);
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
```

Comment:
- [ ] This project's design should be similar to the figma design as mentioned in the requirement.
- You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- Currently, your design has the following problems:
- The margins of `p` tag is not respected as in Figma design, `p` tag has 24 pixels margin on both sides while your margin is more than that.
- The color of `p` tag and icons are not similar to the Figma design.
your design
![qw](https://user-images.githubusercontent.com/96683843/212147821-c35c95ef-4595-4e58-8fe1-708e6681691f.jpg)

figma design
 
![fg](https://user-images.githubusercontent.com/96683843/212147893-4e6218c4-d1be-49a3-b4e4-4a479416551e.jpg)